mavis/summary/summary.py

```diff
@@ -239,15 +240,15 @@ def filter_by_evidence(
                 bpp.break2_split_reads < filter_min_split_reads,
                 all([
                     bpp.event_type != SVTYPE.INS,
-                    bpp.break2_split_reads_forced + bpp.break1_split_reads_forced < filter_min_linking_split_reads
+                    bpp.linking_split_reads < filter_min_linking_split_reads
                 ]),
                 all([
                     bpp.event_type == SVTYPE.INS,
                     bpp.flanking_pairs < filter_min_linking_split_reads,
```

Comment:
Shouldn't this be 
`bpp.flanking_pairs < filter_min_flanking_reads,`

Reply:
no, b/c that filter is specific to call by flanking. For insertions, the untemplated sequence may be greater than a single read and therefore its possible that the split reads may not align to both breakpoints. In such a case the linking evidence would be by flanking pairs instead

Reply:
oh right! I think you might have mentioned that to me before